spec/services/founders/mark_as_exited_service_spec.rb

```diff
@@ -10,8 +10,8 @@
     it 'creates a new startup in the same level and mark the founder as exited' do
       old_startup = student.startup
 
-      expect { subject.execute }.to change { student.reload.exited }.from(false).to(true)
-      expect(student.reload.startup).not_to eq(old_startup.id)
+      expect { subject.execute }.to change { student.reload.startup.exited_at }.from(nil)
+      expect(student.startup).not_to eq(old_startup.id)
```

Comment:
The `startup` object is being compared against the value of `.id`, which will _never_ match. The old spec was also busted here.